### DISCUSSION TITLE
Any chance of a release soon for urllib3 >= 2.0.0 support?

### DISCUSSION THREAD

**Discussion by rgilton** [cobrateam/splinter#1195] [Q&A]
Hi,

I'm wondering if there is a chance that there might be a release of this package soon?  Looks like support for urllib3 > 2.0.0 is in the main branch, but not released yet.  We have a dependency that requires urllib3, so we can't install the latest release alongside splinter :disappointed: 

Thanks,

Rob

**Comment by fsouza** [comment]
We should be able to do it. @jsfehler was there anything you wanted to ship/merge before we cut a new release?
